src/sentry/search/django/backend.py

```diff
@@ -273,7 +273,7 @@ def _build_queryset(
         )
         return queryset
 
-    def query(self, project, count_hits=False, **kwargs):
+    def query(self, project, count_hits=False, paginator_options={}, **kwargs):
```

Comment:
Don't get into the bad habit of using mutable function arguments: http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments

Reply:
wow i had no idea ted

Reply:
good lord

Reply:
🙃